components/patina_adv_logger/README.md

```diff
@@ -1,32 +1,93 @@
    This allows the logger to adopt the buffer and record its address for later protocol publication.
-3. Discover or provision an Advanced Logger buffer.
-   Note: Platform firmware typically passes the Advanced Logger HOB from a prior boot phase.
-4. Dispatch the `AdvancedLoggerComponent` through Patina so it can install the Advanced Logger protocol via boot
-   services.
+3. Register the advanced logger component (`AdvancedLoggerComponent`) to be dispatched by the Patina DXE Core so it
+   can install the Advanced Logger protocol via boot services.
 
-## Memory Log Behavior
+#### Example
 
-The crate stores records in a shared buffer that begins with an `ADVANCED_LOGGER_INFO` header.
+```rust
+use patina_dxe_core::*;
+use patina::{log::Format, serial::uart::UartNull};
+use patina_adv_logger::{component::AdvancedLoggerComponent, logger::AdvancedLogger};
 
-- Aligned entries follow the header.
-- Each entry records the boot phase identifier, EFI debug level mask, timestamp counter, and message bytes.
+use log::LevelFilter;
+use core::ffi::c_void;
 
-## Parser Support
+static LOGGER: AdvancedLogger<UartNull> = AdvancedLogger::new(
+   Format::Standard, // How logs are formatted
+   &[("allocations", LevelFilter::Off)], // set custom log levels per module
+   log::LevelFilter::Info, // Default log level
+   UartNull { }, // Serial writer instance
+);
+
+struct ExamplePlatform;
+
+impl ComponentInfo for ExamplePlatform {
+   fn components(mut add: Add<Component>) {
+      add.component(AdvancedLoggerComponent::<UartNull>::new(&LOGGER));
+   }
+}
+
+#[cfg_attr(target_os = "uefi", unsafe(export_name = "efi_main"))]
+pub extern "efiapi" fn _start(physical_hob_list: *const c_void) -> ! {
+   log::set_logger(&LOGGER).map(|()| log::set_max_level(log::LevelFilter::Trace)).unwrap();
+
+   // SAFETY: The physical_hob_list pointer is assumed to be valid as it is provided to the entry_point from an
+   // external caller.
+   if let Err(e) = unsafe { LOGGER.init(physical_hob_list) } {
+      log::error!("Failed to find a the Advanced Logger HOB. Cannot write to the memory log.");
+   }
+
+   # loop { }
+}
+```
 
-When the `std` feature is enabled the crate provides a `parser` module for captured log buffers.
+### Platform Integration
 
-It opens the buffer, prints header metadata, and emits log lines with optional level and timestamp context.
-This parser underpins host utilities and remains version-aligned with the memory layout implemented in `memory_log.rs`.
+Currently, the Patina Advanced Logger expects that a log buffer has already been created prior to the Patina DXE Core
+being executed. The location of this buffer is provided via a HOB in the HOB list. So long as it is provided, the
+logger and component will execute as expected. If your platform is a EDKII style platform, a [PEI Core Library](https://github.com/microsoft/mu_plus/blob/d0d305b620baced42adf16b2387af9412fdc0ef9/AdvLoggerPkg/Library/AdvancedLoggerLib/PeiCore/AdvancedLoggerLib.inf)

```

Comment:
nit
```suggestion
logger and component will execute as expected. If your platform is a EDK II style platform, a [PEI Core Library](https://github.com/microsoft/mu_plus/blob/d0d305b620baced42adf16b2387af9412fdc0ef9/AdvLoggerPkg/Library/AdvancedLoggerLib/PeiCore/AdvancedLoggerLib.inf)
```

If it's easy enough to find and replace in these modified files, please do so. Otherwise, feel free to ignore.